PKGBUILD

```diff
@@ -89,18 +98,18 @@ prepare() {
   # Force script incompatible with Python 3 to use /usr/bin/python2
   sed -i '1s|python$|&2|' third_party/dom_distiller_js/protoc_plugins/*.py
 
+  mkdir -p third_party/node/linux/node-linux-x64/bin
+  ln -s /usr/bin/node third_party/node/linux/node-linux-x64/bin/
```

Comment:
Node.js is only used if we have `optimize_webui` enabled, but we currently have `optimize_webui=false` in ungoogled-chromium's flags.gn. Do you want `optimize_webui` to be enabled?

Reply:
after reading https://chromium.googlesource.com/chromium/src/+/HEAD/docs/optimizing_web_uis.md I don't see a reason to have ```optimize_webui``` false, why did you do that? to reduce compile times?